portal/config/config.py

```diff
@@ -93,8 +93,10 @@ class BaseConfig(object):
     LOG_LEVEL = 'DEBUG'
 
     OAUTH2_PROVIDER_TOKEN_EXPIRES_IN = 4 * 60 * 60  # units: seconds
-    SS_TIMEOUT = 60 * 60  # seconds for session cookie, reset on ping
-    PERMANENT_SESSION_LIFETIME = SS_TIMEOUT
+    DEFAULT_INACTIVITY_TIMEOUT = 30 * 60  # default inactivity timeout
+    PERMANENT_SESSION_LIFETIME = 60 * 60  # defines life of redis session
```

Comment:
This also sets the lifetime of the cookie FYI
http://flask.pocoo.org/docs/1.0/config/#PERMANENT_SESSION_LIFETIME

Reply:
Yes, but that's fine.  You'll note the cookie used to retain the /settings value is given a different lifetime.